corona-warn-app/cwa-app-android
# Uqs error handling / dialog clenaup (EXPOSUREAPP-9777)

This PR addresses ExpsoureApp-9777.
Dialog adjustments.

How to test:
- Open the Universal Scanner
- Scan an invalid QR code (can be found in the ticket)
- Check the dialog and dialog navigation

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9777
